cuml/svm_demo.ipynb

```diff
@@ -177,6 +184,160 @@
     "cumlSVC = cuml.svm.SVC(kernel=kernel, C=C, tol=tol, gamma='scale', cache_size=2000)\n",
     "cumlSVC.fit(X_train.astype(np.float32), y_train)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Regression\n",
```

Comment:
one nitpick, since above it says `Support vector classification` maybe we can change this to `Support vector regression`

Reply:
done